Lib/fontTools/cffLib/__init__.py

```diff
@@ -2310,6 +2304,7 @@ def decompileAllCharStrings(self, progress):
 				progress.increment(0)  # update
 			i = i + 1
```

Comment:
I am not, and I doubt anyone is. If there's a safe way to deprecate it then I'm fine with that.